src/main/java/io/stargate/sgv2/jsonapi/config/OperationsConfig.java

```diff
@@ -280,6 +280,12 @@ interface ConsistencyConfig {
   @WithDefault("false")
   boolean vectorizeEnabled();
 
+  /**
+   * @return Flag to extend error response with additional information.
+   */
+  @WithDefault("false")
```

Comment:
nit: should probably be ordered alphabetically same as other settings.
(but it sounds like we'll be grouping Feature flags together soon anyway).